src/js/components/ColumnChooserMenu.js

```diff
@@ -139,16 +159,39 @@ export default class ColumnChooserMenu extends React.Component<Props> {
                 Deselect All
               </Paragraph>
             </ControlListItem>
-            {columns.map((c) => (
-              <ColumnListItem key={`${c.name}-${c.type}`}>
-                <Checkbox
-                  label={c.name}
-                  checked={c.isVisible}
-                  onChange={(e) => this.onColumnClick(e, c)}
+            <ControlListItem>
+              <div className="search-input">
+                <SearchInput
+                  id="column-search"
```

Comment:
```suggestion
                  autoFocus
```

Looks like the id isn't necessary.

And I think an autoFocus would be helpful here.